### ISSUE TITLE
saving does not work if file extension is missing

### ISSUE THREAD

**Issue by Fer60** [graasp/graasp-desktop#155]
After having previewed a space in the desktop viewer I can no longer save it for offline use. The message "Error
There was an error saving the space" appears.

I had saved a space previously and deleted this, so it did work once.

**Comment by Fer60** [comment]
The link to the ILS I tried to save is 7slvq0

**Comment by juancarlosfarah** [comment]
@Fer60, I have been able to reproduce and I can confirm that this is a bug. Thanks for reporting.

In this case, our extension matcher is failing to recognise the fileType of the background image, which for some reason is returning without an extension @ahgentil. Can you look into it from the API's perspective? URL: http://api.graasp.eu/spaces/5d3f0cb5fd555f7e3fbe7ee2/offline-structure

```
"image": {
"backgroundUrl": "//graasp.eu/backgrounds/5d3f0cb5fd555f7e3fbe7ee2/orig_undefined",
"thumbnailUrl": "//graasp.eu/pictures/5d3f0cb5fd555f7e3fbe7ee2/orig_ac1b66971e060dcea766b379cb2ff70277b0c5f0.jpeg"
},
```

The error we're seeing on the desktop side is:

```
[2019-08-06 15:19:12.701] [error] TypeError: Cannot read property '1' of null
    at getExtension (/Users/juancarlosfarah/code/graasp/native/graasp-desktop/public/app/utilities.js:25:37)
    at downloadSpaceResources (/Users/juancarlosfarah/code/graasp/native/graasp-desktop/public/app/download.js:58:21)
    at EventEmitter.<anonymous> (/Users/juancarlosfarah/code/graasp/native/graasp-desktop/public/app/listeners/saveSpace.js:46:25)
```

I will ensure that we can handle this without failing and include the fix in today's release.

**Comment by juancarlosfarah** [comment]
@Fer60, this should be solved in v0.11.5. Thanks again for reporting!

**Comment by juancarlosfarah** [comment]
Caused by the following issue in Graasp / Graasp API.

https://github.com/react-epfl/graasp-api/issues/205
